File: Projeto/Ajax-Uploader/MVC-CSharp/Controllers/HomeController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace MVCCSharp.Controllers
{
	[HandleError]
	public class HomeController : Controller
	{
		public ActionResult Index()
		{
            ViewData["Message"] = "MVC File Upload like GMail";

			return View();
		}
	}
}

File: Projeto/Ajax-Uploader/MVC-VB/Controllers/HomeController.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports System.Web.Mvc

Namespace MVCVB.Controllers
	<HandleError()> _
	 Public Class HomeController
		Inherits Controller
		Public Function Index() As ActionResult
			ViewData("Message") = "MVC File Upload like GMail"

			Return View()
		End Function
	End Class
End Namespace